BParkingNano/python/BToKLL_cff.py

```diff
@@ -0,0 +1,111 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.BParkingNano.common_cff import *
+
+electronPairsForKee = cms.EDProducer(
+    'DiElectronBuilder',
+    src = cms.InputTag('electronsForAnalysis', 'SelectedElectrons'),
+    transientTracksSrc = cms.InputTag('electronsForAnalysis', 'SelectedTransientElectrons'),
+    lep1Selection = cms.string('pt > 1.5 && (userInt("isPF") == 1 || userFloat("unBiased") >= 3 )'),
+    lep2Selection = cms.string('pt > 0.5 && (userInt("isPF") == 1 || userFloat("unBiased") >= -4)'),
+    preVtxSelection = cms.string(
+        'abs(userCand("l1").vz - userCand("l2").vz) <= 1. && mass() < 5 '
+        '&& mass() > 0 && charge() == 0 && userFloat("lep_deltaR") > 0.03'
+    ),
+    postVtxSelection = cms.string('userFloat("sv_chi2") < 998 && userFloat("sv_prob") > 0'),
+)
+
+BToKee = cms.EDProducer(
+    'BToKLLBuilder',
+    dileptons = cms.InputTag('electronPairsForKee'),
+    leptonTransientTracks = electronPairsForKee.transientTracksSrc,
+    kaons = cms.InputTag('tracksBPark', 'SelectedTracks'),
+    kaonsTransientTracks = cms.InputTag('tracksBPark', 'SelectedTransientTracks'),
+    beamSpot = cms.InputTag("offlineBeamSpot"),
+    kaonSelection = cms.string(''),
+    preVtxSelection = cms.string(
+        'pt > 3. && userFloat("min_dr") > 0.01 '
+        '&& mass < 6 && mass > 4.5'
+        ),
+    postVtxSelection = cms.string(
+        'userInt("sv_OK") == 1 && userFloat("sv_prob") > 0.00000001 '
+        '&& userFloat("cos_theta_2D") >= 0'
+    )
+)
+
+muonPairsForKmumu = cms.EDProducer(
+    'DiMuonBuilder',
+    src = cms.InputTag('muonTrgSelector', 'SelectedMuons'),
+    transientTracksSrc = cms.InputTag('muonTrgSelector', 'SelectedTransientMuons'),
+    lep1Selection = cms.string(''), #FIXME
+    lep2Selection = cms.string(''),
+    preVtxSelection = electronPairsForKee.preVtxSelection,
+    postVtxSelection = electronPairsForKee.postVtxSelection,
+)
+
+BToKmumu = cms.EDProducer(
+    'BToKLLBuilder',
+    dileptons = cms.InputTag('muonPairsForKmumu'),
+    leptonTransientTracks = muonPairsForKmumu.transientTracksSrc,
+    kaons = BToKee.kaons,
+    kaonsTransientTracks = BToKee.kaonsTransientTracks,
+    beamSpot = cms.InputTag("offlineBeamSpot"),
+    kaonSelection = cms.string(''),
+    # This in principle can be different between electrons and muons
+    preVtxSelection = cms.string(
+        'pt > 3. && userFloat("min_dr") > 0.01 '
+        '&& mass < 6 && mass > 4.5'
+        ),
+    postVtxSelection = cms.string(
```

Comment:
again, minor comment:
in order not to change the cuts wrt to current master, it should be
userFloat("sv_prob") > 0.001